pkg/apis/sources/v1alpha2/sinkbinding_conversion.go

```diff
@@ -18,17 +18,43 @@ package v1alpha2
 
 import (
 	"context"
-	"fmt"
 
+	"knative.dev/eventing/pkg/apis/sources/v1beta1"
 	"knative.dev/pkg/apis"
+	duckv1alpha1 "knative.dev/pkg/apis/duck/v1alpha1"
+	duckv1beta1 "knative.dev/pkg/apis/duck/v1beta1"
 )
 
-// ConvertTo implements apis.Convertible
-func (source *SinkBinding) ConvertTo(ctx context.Context, sink apis.Convertible) error {
-	return fmt.Errorf("v1alpha2 is the highest known version, got: %T", sink)
+// ConvertTo implements apis.Convertible.
+// Converts source (from v1alpha2.SinkBinding) into v1alpha1 or v1beta1.
```

Comment:
I checked the interface
https://github.com/knative/pkg/blob/3ba0019af6be46357ca645f457be0f613b760f0c/apis/interfaces.go#L39
Looks like convert to just convert to/from "higher" versions. 
Looks like the comments should be `Converts source (from v1alpha2.SinkBinding) into v1beta1.` or `Converts source (from v1alpha2.SinkBinding) into higher versions.`

Reply:
Done, thanks!